auth-web/src/components/auth/account-settings/account-info/AccountInfo.vue

```diff
@@ -386,24 +396,32 @@ export default class AccountInfo extends Mixins(
       businessSize: this.currentOrganization?.businessSize || ''
     }
   }
+  private get isBusinessInfoIncomplete (): boolean {
+    return this.isBusinessAccount &&
+        !(this.accountDetails?.businessSize && this.accountDetails?.businessType);
+  }
+  private get isAddressInfoIncomplete (): boolean {
+    return this.currentOrgAddress ? Object.keys(this.currentOrgAddress).length === 0 : true;
+  }
+
   private async setup () {
     this.setAccountDetails()
     await this.syncAddress()
     // show this part only account is not anon
     if (!this.anonAccount) {
       this.originalAddress = this.currentOrgAddress
-      if (Object.keys(this.currentOrgAddress).length === 0) {
-        this.isCompleteAccountInfo = false
-        this.errorMessage = this.isAddressEditable
-          ? 'Your account info is incomplete. Please enter your address in order to proceed.'
-          : 'This accounts profile is incomplete. You will not be able to proceed until an account administrator ' +
-            'entered the missing information for this account.'
-        this.$refs.editAccountForm?.validate() // validate form fields and show error message
-        // SBTODO create a method in child comp
-        this.$refs.mailingAddress?.triggerValidate() // validate form fields and show error message for address component from sbc-common-comp
+      const getWarningMessage = (condition: Boolean) => condition
+        ? 'This account info is incomplete. You will not be able to proceed until an account administrator enters the missing information for this account.'
```

Comment:
Linting issue, break this into two lines

Reply:
Fixed